.github/workflows/backend-build.yml

```diff
@@ -0,0 +1,60 @@
+name: Backend Build Tests
+
+on:
+  push:
+    branches:
+      - main
+      - master
+      - github_actions # TODO delete this if ever merged to master/main
+  pull_request:
+    branches:
+      - main
+      - master
+      - github_actions # TODO delete this if ever merged to master/main
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.17.0'
```

Comment:
we are at 1.17.3 now, if it's available here :)

Reply:
whoops just saw - ill make a pr to fix

Reply:
https://github.com/kubernetes-sigs/kpng/pull/98